rules/windows/persistence_run_key_and_startup_broad.toml

```diff
@@ -190,8 +190,9 @@ registry where host.os.type == "windows" and registry.data.strings != null and
 
     /* Slack */
     (
-      process.code_signature.trusted == true and process.code_signature.subject_name : "Slack Technologies, Inc." and
-      process.name : "slack.exe" and registry.data.strings : (
+      process.code_signature.trusted == true and process.code_signature.subject_name : (
```

Comment:
Since signatures are case sensitive, it would be more stringent and accurate to us `in` and `==`

Reply:
Some companies still have variations on their certificates, e.g., Nvidia has both `Nvidia Corporation` and `NVIDIA Corporation` as the subjects. Looking for the exact case could mean having more FPs here

Reply:
but imitation bypasses become easier without enforcing sensitivity

Reply:
Though I agree, the inconsistency of signatures by the same vendor is painful